Breeding-Insight/bi-api
# BI-1721 - Show All - Breeding Methods table

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1721

When trying to enable system breeding methods, and a given program has previously created a custom breeding method, an error was thrown stating "Breeding method is not allowed to be edited".  This was due to the program custom breeding method being included in the check for editability when it should not have been.



# Dependencies
bi-web: [bug/BI-1721](https://github.com/Breeding-Insight/bi-web/pull/303)

# Testing
1. For a given program, on the Breeding Methods config page
2. Enable a subset of system methods
3. Create a new program custom breeding method
4. Enable all system methods, and ensure this succeeds


# Checklist:
